tests-beta/android/MASVS-RESILIENCE/MASTG-TEST-0x41.md

```diff
@@ -0,0 +1,26 @@
+---
+platform: android
+title: Testing if StrictMode is enabled
```

Comment:
When writing test titles, imagine that they already start with "Testing for/if", but don't add that to the title. With this suggestion, I'm trying to focus on the observable behavior (violation logs) rather than the internal configuration/presence/"enabled" in the code, which we could reserve for the static version of the test.

```suggestion
title: Logging of StrictMode Violations
```